Turn Github Issue templates into forms #361

plon-Susk7 opened the pull request:
---
name: Feature Enhancement
about: turning github issue templates into forms
title: Turn Github Issue templates into forms #361
labels: enhancement
assignees: ''

---

**Describe the PR**
made yaml files for issue templates and deleted old markdown template files

**Steps to test the PR**
Not sure how to test my PR, but the official syntax was followed.

**Expected behavior**
The issue template now will be more user friendly and easy to fill.

**Screenshots**
N/A


**Additional context**
N/A

plon-Susk7 commented:
@dennyabrain @aatmanvaidya please review my PR, thanks :)

aatmanvaidya commented:
@plon-Susk7 thank you for the work, give us some time to review, we will get back to you!

aatmanvaidya commented:
Hi @plon-Susk7 , gave it a test by creating issues on your demo github repo 

some suggestions

in the **Issue: Feature Request**
1. Can you make the question - "Describe alternatives you've considered" not compulsory, because it not necessarily everyone would have considered some alternative solution. 

in the **Issue: Bug Report**
1. In the question - "What browsers are you seeing the problem on?", can you also add a dropdown option called "Other", because it might happen (although highly unlikely), that some users are using other browsers that what is there in the options.
2. Just curious, why is the last question about accepting the Code of Conduct? Is it automatically there by GitHub forms or did you add it? 
3. Once I submit the bug report issue, why does **1 task done** bubble pop-up, like in the image below. Again just curious, it might GitHub automatically adding that, I am just thinking out loud. 
![Screenshot from 2023-10-10 20-18-23](https://github.com/tattle-made/Uli/assets/56875084/42f00c4d-c90c-497c-93d3-2d4a92be044c)

In the **Issue: QA Test Checklist**
1. Can we add just like a text box, where a user can also request QA testing for something else then a feature, it could be say a problem in viewing the image logo in different browsers, it could be some text issue as well, we never know. 


These are just my thoughts, @dennyabrain please feel free to correct me!

@plon-Susk7 good work!

plon-Susk7 commented:
Hey @aatmanvaidya ,

**Feature Request :** Done


**Bug Report:** 
1. Done
2. Removed Code of conduct, it was there in the documentation I forgot to remove it.
3. It's a default thing by github as I have added no extra code for it.


**QA Test Checklist:** Could you please elaborate it a more.

plon-Susk7 commented:
Nevermind got it @aatmanvaidya . Pushing my changes to both repo.

aatmanvaidya commented:
thanks @plon-Susk7 

@dennyabrain tested this PR, things look good now, its a green light to merge from my side.

aatmanvaidya commented:
Hi @plon-Susk7 , some changes to the **Issue: Bug Report** are required. 

As you can see in the image below. 

![Screenshot from 2023-10-11 10-51-23](https://github.com/tattle-made/Uli/assets/56875084/eedb0389-7a17-4b16-97b3-726d7a8c4fe5)

In the placeholder text in **Desktop** and **Smartphone** which says "OS: [e.g iOS] is not proper, as it says the same for Desktop, so we might have to change that

So since, we already have a drop down for the browser - What browsers are you seeing the problem on?
Can we create a similar one for devices, so the question becomes - "Which operating system are you on?" and the options will be - Windows, MacOS, Linux, Android, iOS

and then we can have a text box asking more information about their system setup, something called - "More information about your setup"

plon-Susk7 commented:
Hi @aatmanvaidya, I have pushed the changes on both repos.

aatmanvaidya commented:
great

@dennyabrain please suggest any more changes if required.

dennyabrain commented:
lgtm. merging.
